caravel/assets/spec/javascripts/sqllab/TabbedSqlEditors_spec.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { TabbedSqlEditors } from '../../../javascripts/SqlLab/components/TabbedSqlEditors';
+import { Tab } from 'react-bootstrap';
+import { shallow } from 'enzyme';
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+import { initialState } from './common';
+
+describe('TabbedSqlEditors', () => {
```

Comment:
it would be great to assert that the correct sub-components are found in this component. we should assert it renders a `<Tab />` and a `<SqlEditor />` for each tab. each tab should have a `<DropdownButton />` and each DropdownButton should have 3 `<MenuItem />` components. i see an opportunity to make the `TabbedSqlEditors` component smaller by pulling parts out into their own components, like a `Tabs` component and a`SQLEditorActionsButton` component.